### PR TITLE
CCD-4035 - PR to enable ignored FTS

### PR DESCRIPTION
**Pull request by marianadpereira**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-4035

### Change description ###

Enabling FTs and including new fields.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_CNP
Comment (marianadpereira) [inline Jenkinsfile_CNP:16]: This needs to be reverted before merging.

File: charts/aac-manage-case-assignment/values.preview.template.yaml
Comment (marianadpereira) [inline charts/aac-manage-case-assignment/values.preview.template.yaml:17]: This needs to be reverted before merging.

File: charts/aac-manage-case-assignment/values.preview.template.yaml
Comment (marianadpereira) [inline charts/aac-manage-case-assignment/values.preview.template.yaml:18]: This needs to be reverted before merging.

**Review by RebeccaBaker — APPROVED** [review]
LGTM - pr build failing though

File: build.gradle
Comment (mattnayler) [inline build.gradle:379]: Should be updated with proper release number once hmcts/ccd-test-definitions#61 is released.

File: Jenkinsfile_CNP
Comment (mattnayler) [inline Jenkinsfile_CNP:82]: I think this can only be safely set to the PR/BRANCH_NAME version of the MCA APIs when the Data Store API is running in isolation.  Otherwise the tests risk being affected by parallel pipelines running in against any CCD Repo that re-publishes the common Test Defintions.

**Review by mattnayler — CHANGES_REQUESTED** [review]

File: Jenkinsfile_CNP
Comment (mattnayler) [reply Jenkinsfile_CNP:83]: 👍 CALLBACK_FTA_ENABLED process should now resolve this

**Review by mattnayler — APPROVED** [review]
Approved subject to remaing green after applying patch in #463

File: Jenkinsfile_CNP
Comment (marianadpereira) [reply Jenkinsfile_CNP]: Solved in [cdd8c09](https://github.com/hmcts/aac-manage-case-assignment/pull/455/commits/cdd8c09005cadb5af36a9cf1186baae088ffd5fa)

File: charts/aac-manage-case-assignment/values.preview.template.yaml
Comment (marianadpereira) [reply charts/aac-manage-case-assignment/values.preview.template.yaml]: Solved in [cdd8c09](https://github.com/hmcts/aac-manage-case-assignment/pull/455/commits/cdd8c09005cadb5af36a9cf1186baae088ffd5fa)

File: charts/aac-manage-case-assignment/values.preview.template.yaml
Comment (marianadpereira) [reply charts/aac-manage-case-assignment/values.preview.template.yaml]: Solved in [cdd8c09](https://github.com/hmcts/aac-manage-case-assignment/pull/455/commits/cdd8c09005cadb5af36a9cf1186baae088ffd5fa)

**Pull request merged**
